tsconfig.json

```diff
@@ -15,7 +15,7 @@
       "dom",
       "es5",
       "scripthost",
-      "es2015"
+      "es2019"

```

Comment:
Necessary for `Array.flatMap`. Not sure if this breaks anything, if it does I can revert it since a `flatMap` can easily be transformed into `es2015` compatible code